.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,122 @@
+
+Please, use the Table of Contents to find a specific section of this guide quickly.
+
+#### Table of Contents
+
+[Code of Conduct](#code-of-conduct)
+
+[I Just Have a Question](#i-just-have-a-question)
+
+[Getting Started](#getting-started)
+  * [Join GitHub](#join-github)
+  * [Getting Used to Git](#getting-used-to-git)
+  * [Common Workflow for Contributing to OpenIPSL](#common-workflow-for-contributing-to-openipsl)
+
+[How Can I Contribute?](#how-can-i-contribute-?)
+  * [Getting Started](#getting-started-by-interacting-beginner)
+  * [Reporting New Bugs](#reporting-new-bugs-bug)
+  * [Improving Documentation ](#improving-documentation-memo)
+  * [Improving Models in the Library](#improving-models-in-the-library-rocket)
+
+[The Contributor License Agreement](#the-contributor-license-agreement)
+
+
+## Code of conduct
+
+OpenIPSL and all of its contributors are governed by the [OpenIPSL Code of Conduct](CODE_OF_CONDUCT.md). By participating in this community you are expected to read this code and to abide by it. We expect you to keep our community approachable and respectable.
+
+## I Just Have a Question!!!
+
+If you just have a question, there is no need for you to read this entire document. Use our [Discussions Page](https://github.com/OpenIPSL/OpenIPSL/discussions) to start a question. It is better (and faster) to reach out to the developers by asking a question. In addition, discussions can further be filed as an issue, so you can easily start your contribution by starting a discussion.
+
+> **Note:** PLEASE DON'T FILE AN ISSUE TO ASK A QUESTION.
+
+## Getting Started
+
+There are many different resources that can be used for getting started with the different subjects that involve a contribution to OpenIPSL. There are a few things that you, as user, might want to check beforehand. 
+
+### Join GitHub
+
+However you decide to contribute to OpenIPSL you should fist create your GitHub account, if you do not have one already. To create your account, just go to this [website](https://github.com/join), fill out the form, complete the captcha and click on Create Account. After that, you will need some material to get started with your git skills. 
+
+### Getting Used to Git
+
+Git is a version control solution that is used to manage the library and its model packages. The following is a list of things that can help you with getting started with Git.
+
+
+* Since you already have a GitHub account at this point, you can start by setting up Git. Useful information about that is found [here](https://docs.github.com/en/get-started/quickstart/set-up-git), but you can also use your web search tool to look for other sources of information.
```

Comment:
```suggestion
* Since you already have a GitHub account at this point, you can start by setting up Git. Useful information about that is found on [GitHub's quick start guide](https://docs.github.com/en/get-started/quickstart/set-up-git), but you can also use your web search tool to look for other sources of information.
```